packages/shared/styles/components/molecules/SfRadio.scss

```diff
@@ -39,6 +40,11 @@
     position: absolute;
     opacity: 0;
   }
```

Comment:
If we stay with the solution below I'd add a top property to move this element away (assistive technology sees this input and puts its own outline on it so that we see both - a little bit of outline and shadow) wdyt? :)